### DISCUSSION TITLE
Should submissions for optimizers (scipy.minimize) be submitted here, or in jaxopt?

### DISCUSSION THREAD

**Discussion by ghost** [jax-ml/jax#10301] [General] [answered]
Thanks for the all the amazing work! I have some free time ahead of me, so I was interested in adjusting my implementations for different optimizers (based on Nocedal & Wright) and I was wondering whether I should submit them here, or in jaxopt.

**Comment by jakevdp** [comment, marked as answer]
I would suggest any new submissions going to `jaxopt`. The minimizers in `jax.scipy.minimize` are somewhat constrained by the need to match the `scipy.minimize` API, and long-term we're considering dropping it in favor of `jaxopt`.

**Reply by ghost** [reply]
Great, thank you!
